proposals/dao/fip_24.js

```diff
@@ -0,0 +1,108 @@
+  const adminSigner = ethers.provider.getSigner(timelockAddress);
```

Comment:
Seems like a convoluted way to get the signer we're supposed to be using here - why not from hardhat's hre?